src/strings/index.tsx

```diff
@@ -646,6 +646,9 @@ const strings = new LocalizedStrings({
     TOTAL_SEEDS_TESTED: 'Total Seeds Tested',
     APPLY_RESULT_QUESTION: 'Do you want to apply this result to the accession?',
     APPLY_RESULT: 'Apply Result',
+    SEEDS_TESTED_ERROR: 'Seeds tested should be less than remaining seeds',
+    SEEDS_GERMINATED_ERROR: 'Germinated amount should be less than tested amount.',
```

Comment:
both these messages should probably say 'should not be more than' instead of 'should be less than' (since they don't have to be less than, they can be equal to)

Reply:
Yes, you are right, it could be equal, I will change it. I took the message from the figma file but this make sense